src/components/more-from-xl-blog/more-from-xl-blog.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../../components/card";
+
+import * as styles from './more-from-xl-blog.module.scss';
+
+const MoreFromXlBlog = ({ data, refMoreFrom }) => {
```

Comment:
I think that the name of the component is confusing for the place where it is located. Correct me if I'm wrong, but all components into `components` are to use as a common component on every page right? So, can `MoreFromXlBlog` be used on all pages?

Reply:
Done